packages/story-editor/src/components/style/textSelectionIcon.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { __ } from '@googleforcreators/i18n';
+import { Icons } from '@googleforcreators/design-system';
+
+function TextSelectionIcon() {
+  return <Icons.LetterT title={__('Text selection', 'web-stories')} />;
```

Comment:
Wait, is this built-in? It actually correctly renders the SVG with `<svg ...><title>{title}</title>...</svg>`. That's amazing!